package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@superfluid-finance/tokenlist-repo",
   "private": true,
-  "version": "2.0.3",
+  "version": "2.1.0",
```

Comment:
We could keep this version in sync with the token list's, not the npm package's.